Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -305,6 +306,16 @@ void IOWindow::CreateMainLayout()
   m_functions_combo->addItem(QStringLiteral("max"));
   m_functions_combo->addItem(QStringLiteral("clamp"));
 
+  m_variables_combo = new QComboBoxWithMouseWheelDisabled(this);
+  m_variables_combo->addItem(tr("User Variables"));
+  m_variables_combo->setToolTip(
+      tr("User defined variables usable in the control expression.\nYou can use them to save or "
```

Comment:
should probably be something more like.

"You can use them to save or retrieve values, and can even share states between inputs and outputs.

The double or, while technically correct in this particular case, reads weird.